extensions/ql-vscode/src/vscode-tests/cli-integration/remote-queries/variant-analysis-monitor.test.ts

```diff
@@ -93,13 +91,12 @@ describe('Variant Analysis Monitor', async function() {
 
       it('should mark as failed locally and stop monitoring', async () => {
         const result = await variantAnalysisMonitor.monitorVariantAnalysis(variantAnalysis, cancellationToken);
-        variantAnalysis = result.variantAnalysis;
 
         expect(mockGetVariantAnalysis.calledOnce).to.be.true;
         expect(result.status).to.eql('Failed');
         expect(result.error).to.eql(`Variant Analysis has failed: ${mockFailedApiResponse.failure_reason}`);
-        expect(variantAnalysis.status).to.equal(VariantAnalysisStatus.Failed);
-        expect(variantAnalysis.failureReason).to.equal(processFailureReason(mockFailedApiResponse.failure_reason as VariantAnalysisFailureReason));
+        expect(result?.variantAnalysis?.status).to.equal(VariantAnalysisStatus.Failed);
+        expect(result?.variantAnalysis?.failureReason).to.equal(processFailureReason(mockFailedApiResponse.failure_reason as VariantAnalysisFailureReason));
```

Comment:
I don't think the `result` can be nullable here, so I think it would be clearer to just add the safe navigation operator only to the `result.variantAnalysis`

Reply:
You're right. I've addressed this [here](https://github.com/github/vscode-codeql/pull/1559/commits/644a83d6d8fe9e23ddb982b84fdcec5117440a78).